dhcpv4/bsdp/bsdp.go

```diff
@@ -316,11 +316,11 @@ func InformSelectForAck(ack dhcpv4.DHCPv4, replyPort uint16, selectedImage BootI
 	d.AddOption(dhcpv4.Option{
 		Code: dhcpv4.OptionParameterRequestList,
 		Data: []byte{
-			dhcpv4.OptionSubnetMask,
-			dhcpv4.OptionRouter,
-			dhcpv4.OptionBootfileName,
-			dhcpv4.OptionVendorSpecificInformation,
-			dhcpv4.OptionClassIdentifier,
+			byte(dhcpv4.OptionSubnetMask),
```

Comment:
whoops, sorry, this must have been broken by my last commit, where I didn't run `go test` in the `bsdp` subpackage.. Which I guess is what led to https://github.com/insomniacslk/dhcp/pull/7 :)